goalc/compiler/compilation/Asm.cpp

```diff
@@ -125,6 +125,18 @@ Val* Compiler::compile_asm_ret(const goos::Object& form, const goos::Object& res
   return get_none();
 }
 
+Val* Compiler::compile_asm_fnop(const goos::Object& form, const goos::Object& rest, Env* env) {
```

Comment:
Usually I name the `compile_xxx` function after the name of the GOAL keyword, not the instruction. So this would be `compile_asm_nop_vf`